packages/backend/src/telemetry/service.ts

```diff
@@ -6,20 +6,28 @@ import {
   PeriodicExportingMetricReader
 } from '@opentelemetry/sdk-metrics'
 
-import { ConvertError, RatesService, isConvertError } from '../rates/service'
+import { RatesService, isConvertError } from '../rates/service'
 import { ConvertOptions } from '../rates/util'
 import { BaseService } from '../shared/baseService'
+import { privacy } from './privacy'
 
 export interface TelemetryService {
-  shutdown(): void
-  getOrCreateMetric(name: string, options?: MetricOptions): Counter
-  getOrCreateHistogramMetric(name: string, options?: MetricOptions): Histogram
-  getInstanceName(): string | undefined
-  getBaseAssetCode(): string
-  getBaseScale(): number
-  convertAmount(
-    convertOptions: Omit<ConvertOptions, 'exchangeRate' | 'destinationAsset'>
-  ): Promise<bigint | ConvertError>
+  shutdown(): Promise<void>
+  incrementCounter(
+    name: string,
+    amount: number,
+    attributes?: Record<string, unknown>
+  ): void
+  incrementCounterWithAmount(
+    name: string,
+    convertOptions: Pick<ConvertOptions, 'sourceAmount' | 'sourceAsset'>,
```

Comment:
```suggestion
    amount: Pick<ConvertOptions, 'sourceAmount' | 'sourceAsset'>,
```

Surely we want to make it clear that this is an amount coming in here?

Reply:
yeah this should have been `amount`. just for extra context, an underlying call (`convertAmount`) uses `convertOptions` which is where this originally came from. the `incrementCounterWithAmount` implementation uses `amount` but i forgot to change this one.